#include <iostream>
#include <exception>


class helpException: public std::exception
{

public:

  helpException(const std::string m="Help Called"):msg(m){}
  ~MyException(void);
  const char* what(){return msg.c_str();}

private:
  std::string msg;

};

